rosidl_typesupport_introspection_c/resource/srv__introspection_type_support.h.template

```diff
@@ -0,0 +1,29 @@
+// generated from rosidl_typesupport_introspection_c/resource/srv__introspection_type_support.h.template
+
+@#######################################################################
+@# EmPy template for generating <srv>__introspection_type_support.h files
+@#
+@# Context:
+@#  - spec (rosidl_parser.MessageSpecification)
+@#    Parsed specification of the .srv file
+@#  - get_header_filename_from_msg_name (function)
+@#######################################################################
+@
+@{
+header_guard_parts = [
+    spec.pkg_name, "srv",
```

Comment:
Please use single quotes for all Python strings which don't contain single quotes within the string.